EarlGrey/Additions/XCTestCase+GREYAdditions.m

```diff
@@ -218,13 +179,6 @@ - (void)grey_invokeTest {
       gCurrentExecutingTestCase = self;
       [self grey_setStatus:kGREYXCTestCaseStatusUnknown];
 
-      // We create a new, empty, outputs directory for the test method prior to its invocation.
```

Comment:
OK. I kept this section deleted but put the other ones you commented on back in.